examples/gameserver.yaml

```
@@ -88,15 +88,19 @@ spec:
   #
   # [Stage:Alpha]
   # [FeatureFlag:CountsAndLists]
-  # Lists provides the configuration for generic (player, room, session, etc.) tracking features.
+  # Counts and Lists provides the configuration for generic (player, room, session, etc.) tracking features.
   # Commented out since Alpha, and disabled by default
+  # counters:
+  #   gameCounter:
```

Nit:

```suggestion
  #   games
```
Adding "Counter" the name seems redundant to me, since it's already under `counters`. WDYT?

Same with `playerList` below.